doc/en/user/source/community/cog/cog.rst

```diff
@@ -0,0 +1,134 @@
+.. _cog_plugin:
+
+COG (Cloud Optimized GeoTIFF) Support
+=====================================
+
+`COG <https://github.com/cogeotiff/cog-spec/blob/master/spec.md>`_ (Cloud Optimized GeoTIFF) is a regular GeoTIFF file, aimed at being hosted on a HTTP file server, whose internal organization is friendly for consumption by clients issuing HTTP GET range requests. 
+The COG module allows to set configuration params to connect to a Cloud GeoTIFF as well as adding JARs to the classpath needed to support that.
+
+Installation
+------------
+
+As a community module, the package needs to be downloaded from the `nightly builds <https://build.geoserver.org/geoserver/>`_,
+picking the community folder of the corresponding GeoServer series (e.g. if working on the GeoServer main development branch nightly
+builds, pick the zip file form ``master/community-latest``).
+
+To install the module, unpack the zip file contents into GeoServer own ``WEB-INF/lib`` directory and restart GeoServer.
```

Comment:
To install the module, unpack the zip file contents into the GeoServer ``WEB-INF/lib`` directory and restart GeoServer.